networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,39 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: add rhel latest repos
+    yum_repository:
+      name: rhel_latest
+      description: taken form https://github.com/redhat-performance/perf-dept/blob/master/repo_files/rhel7-latest.repo
+      baseurl: http://download.devel.redhat.com/rel-eng/latest-RHEL-7/compose/Server/x86_64/os/
+
+  - name: install reqiured pkgs
+    yum: state=present name={{ item }}
+    with_items:
+    - kernel
+    - kernel-devel
+    - gcc
+    - js
+
+  - name: download openonload
+    get_url:
+      url: http://www.openonload.org/download/openonload-201606-u1.3.tgz
```

Comment:
maybe better to leave that as is for future debugging if needed, and currently we need to debug it sometimes.